### ISSUE TITLE
Annotation Overlapping

### ISSUE THREAD

**Issue by haroon-sajjad** [Daniel-KM/Omeka-S-module-Cartography#12]
Annotations are overlapping on multiple images. See the screenshots below:

<img width="1052" alt="Screen Shot 2023-06-18 at 11 51 14 PM" src="https://github.com/Daniel-KM/Omeka-S-module-Cartography/assets/8211383/849804c9-51ec-4883-9041-13af9caca949">
<img width="1052" alt="Screen Shot 2023-06-18 at 11 53 03 PM" src="https://github.com/Daniel-KM/Omeka-S-module-Cartography/assets/8211383/9b8f5b3d-4d45-40c0-b6c8-3ebef14d6b0e">

**Comment by Daniel-KM** [comment]
Yes this is an issue. Normally, the annotation has a target property "oa:hasSelector" that indicates the media, then it should be mapped to the right group in leaflet. It is an issue in last version of the module. It was working in an old version.

**Comment by Daniel-KM** [comment]
This is fixed in version 3.4.4.
